opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -86,13 +90,15 @@ message InstrumentationLibraryMetrics {
 //                       |+-----+                    |
 //                       +---------------------------+
 //
-//-----------------------------------------------------------------------
-// DataPoint is a value of specific type corresponding to a given moment in
-// time. Each DataPoint is timestamped.
-//
-// DataPoint is strongly typed: each DataPoint type has a specific Protobuf message
-// depending on the value type of the metric and thus there are currently 4 DataPoint
-// messages, which correspond to the types of metric values.
+// All DataPoint types have includes three common fields:
+// - Labels are the optional key-value pairs associated with the data point.
```

Comment:
If labels are optional, then its incorrect to say DataPoint must include "three" common field.

Reply:
Corrected "zero or more" is not optional. Zero is accepted.